test-poc/framework/src/main/java/org/keycloak/test/framework/webdriver/ChromeWebDriverSupplier.java

```diff
@@ -1,44 +1,17 @@
 package org.keycloak.test.framework.webdriver;
 
-import org.keycloak.test.framework.injection.InstanceWrapper;
-import org.keycloak.test.framework.injection.LifeCycle;
-import org.keycloak.test.framework.injection.Registry;
-import org.keycloak.test.framework.injection.RequestedInstance;
-import org.keycloak.test.framework.injection.Supplier;
 import org.openqa.selenium.WebDriver;
 import org.openqa.selenium.chrome.ChromeDriver;
 
-public class ChromeWebDriverSupplier implements Supplier<WebDriver, TestWebDriver> {
-
-    @Override
-    public Class<TestWebDriver> getAnnotationClass() {
-        return TestWebDriver.class;
-    }
-
-    @Override
-    public Class<WebDriver> getValueType() {
-        return WebDriver.class;
-    }
-
-    @Override
-    public InstanceWrapper<WebDriver, TestWebDriver> getValue(Registry registry, TestWebDriver annotation) {
-        final var driver = new ChromeDriver();
-        return new InstanceWrapper<>(this, annotation, driver, LifeCycle.GLOBAL);
-    }
-
-    @Override
-    public boolean compatible(InstanceWrapper<WebDriver, TestWebDriver> a, RequestedInstance<WebDriver, TestWebDriver> b) {
-        return true;
-    }
-
-    @Override
-    public void close(WebDriver instance) {
-        instance.quit();
-    }
+public class ChromeWebDriverSupplier extends AbstractWebDriverSupplier {
 
     @Override
     public String getAlias() {
```

Comment:
this should be also defined as abstract in AbstractWebDriverSupplier

Reply:
No, alias is optional, the default name for a supplier will be `class.getSimpleName`